include/volume/volume_cooling_gaussians_crhmc.hpp

```diff
@@ -0,0 +1,448 @@
+#ifndef VOLUME_COOLING_GAUSSIANS_CRHMC_HPP
+#define VOLUME_COOLING_GAUSSIANS_CRHMC_HPP
+
+//#define VOLESTI_DEBUG
+
+#include "volume/volume_cooling_gaussians.hpp"
+#include "preprocess/crhmc/crhmc_problem.h"
+
+////////////////////////////// Algorithms
+
+// Gaussian Anealling
```

Comment:
we can remove this, we have it in the original case but you are creating those new functions especially for the second step, i.e. to sample from non-spherical gaussians.